lib/jsdom/living/node.js

```diff
@@ -191,3 +191,16 @@ module.exports.listOfElementsWithNamespaceAndLocalName = (namespace, localName,
     } });
   });
 };
+
+// https://dom.spec.whatwg.org/#converting-nodes-into-a-node
+// create a fragment (or just return a node for one item)
+exports.createNode = (document, nodes) => {
+  if (nodes.length === 1) {
+    return nodes[0] && nodes[0].nodeType ? nodes[0] : document.createTextNode(String(nodes[0]));
```

Comment:
Why not check for `typeof nodes[0] === 'string' ? document.createTextNode(nodes[0]) : nodes[0]`?

The `String()` is unnecessary as the generated IDL interface file already takes care of it.